06-raster-vector.Rmd

```diff
@@ -146,19 +146,22 @@ However, the line extraction approach is not recommended to obtain values along
 In this case, a better approach is to split the line into many points and then extract the values for these points.
 To demonstrate this, the code below creates `zion_transect`, a straight line going from northwest to southeast of the Zion National Park, illustrated in Figure \@ref(fig:lineextr)(A) (see Section \@ref(vector-data) for a recap on the vector data model):
 
+<!--toDo:jn-->
+<!--fix pipes-->
+
 ```{r 06-raster-vector-11 }
 zion_transect = cbind(c(-113.2, -112.9), c(37.45, 37.2)) |>
   st_linestring() |> 
-  st_sfc(crs = crs(srtm)) |> 
-  st_sf()
+  st_sfc(crs = crs(srtm)) %>%
+  st_sf(geometry = .)
```

Comment:
Why not:

```
  st_sfc(crs = crs(srtm)) %>%
  st_sf(geometry = _)
```

Even if that fails on your computer it should pass on actions. I would be curious to see if it does :crossed_fingers:

Reply:
Aha didn't see that, looking good :+1: